python/lance/pytorch/data.py

```diff
@@ -68,7 +89,7 @@ class LanceDataset(IterableDataset):
     def __init__(
         self,
         root: Union[str, Path],
-        columns: Optional[List[str]] = None,
+        columns: Optional[Union[List[str], Dict[str, str]]] = None,
```

Comment:
what's the dict use case?

Reply:
Oh, i was intent to allow `alias` via columns parameter, like `{"labels": "annotations.category_id",  "boxes": "annotations.bbox"}`. 

Will file another ticket to do it appropriately.

Reply:
Filed https://github.com/eto-ai/lance/issues/178